transformations/transformer_text_generation/README.md

```diff
@@ -0,0 +1,36 @@
+# Transformer-based Text Generation
+We use generative pretrained language model (e.g., GPT-2) to generate next word(s) in a sequence based on preceding word(s).
+
```

Comment:
Hi @jmamou please add your name, email and affiliation. Thanks for your changes.